src/goto-instrument/nondet_volatile.h

```diff
@@ -33,9 +33,9 @@ class optionst;
   " --" NONDET_VOLATILE_VARIABLE_OPT " <variable>\n" \
   "                              makes reads from given volatile variable " \
   "non-deterministic\n" \
-  " --" NONDET_VOLATILE_MODEL_OPT "<variable>:<model>\n" \
+  " --" NONDET_VOLATILE_MODEL_OPT " <variable>:<model>\n" \
   "                              models reads from given volatile variable " \
-  "by a call to the given model"
+  "by a call to the given model\n"
```

Comment:
⛏️  would be good if the help text didn't exceed 80 chars.

Reply:
Ok, added a function `help_entry()` which automatically wraps the text at 80 chars.